example/bond/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "build/index.js",
   "scripts": {
     "build": "tsc",
-    "start": "ts-node ./src/index.ts",
+    "start": "npx ts-node ./src/index.ts",
```

Comment:
This should not be needed... all npm scripts run with the PATH of the installed binaries